gst/nnstreamer/tensor_aggregator/tensor_aggregator.c

```diff
@@ -1064,11 +1070,12 @@ gst_tensor_aggregator_parse_caps (GstTensorAggregator * self,
    * if frames_out=10 and frames_dim=2, then out-dimension is 2:200:2000:1.
    */
   g_assert (self->frames_dim < NNS_TENSOR_RANK_LIMIT);
-  g_assert ((config.info.dimension[self->frames_dim] % self->frames_in) == 0);
-  per_frame = config.info.dimension[self->frames_dim] / self->frames_in;
-
-  config.info.dimension[self->frames_dim] = per_frame * self->frames_out;
+  g_assert ((config.info.info[0].dimension[self->frames_dim] %
+          self->frames_in) == 0);
+  per_frame = config.info.info[0].dimension[self->frames_dim] / self->frames_in;
 
+  config.info.info[0].dimension[self->frames_dim] =
+      per_frame * self->frames_out;
```

Comment:
You do not need to correct this because this is not the topic of this PR.

However, for future refactoring and implementation, please note that assert is not recommended in shared libraries especially if such assertion may fail with given inputs and calls by library users. It should be triggered only by internal bugs.

Reply:
LGTM other than this. You may merge this after it's resolved.

Reply:
Updated. PTAL